### PR TITLE
feat: add ops drippie deployments

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Updates the periphery deploy to also deploy a Drippie contract to be used for standard operational drips.

### CODE REVIEW

File: packages/contracts-bedrock/deployments/11155111-deploy.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (clabby) [inline packages/contracts-bedrock/deployments/11155111-deploy.json]: We don't normally commit these intermediate deployments - any specific reasons?

File: packages/contracts-bedrock/scripts/Artifacts.s.sol
Comment (clabby) [inline packages/contracts-bedrock/scripts/Artifacts.s.sol:1]: Why the changes here?

File: packages/contracts-bedrock/periphery-deploy-config/sepolia.json [hunk 2]
```diff
@@ -1,42 +1,69 @@
...
+  "operationsGelatoDripV1Recipient": "0x03C256F7Ae7518D0fe489F257ab4b928D37CBE16",
+  "operationsGelatoDripV1Value": 1000000000000000000,
+  "operationsGelatoDripV1Interval": 86400,
+  "operationsGelatoDripV1Threshold": 500000000000000000,
+  "operationsSecretsDripV1Delay": 43200,
+  "operationsSecretsDripV1MustExist": "0x565fa8c7daa859353b5b328b97b12c7d66c5832b2a24d4e0f739a65ad266a46f",
+  "operationsSecretsDripV1MustNotExist": "0xbc362b01d69a85dff1793803dde67df1f338f37a36fdc73dddf27283d215e614",
+  "operationsSecretsDripV1Target": "0x03C256F7Ae7518D0fe489F257ab4b928D37CBE16",
+  "operationsSecretsDripV1Value": 1000000000000000000,
+  "operationsSecretsDripV1Interval": 3600,
```
Comment (clabby) [inline packages/contracts-bedrock/periphery-deploy-config/sepolia.json:21]: Any references for this new configuration? Values seem sane, curious if these were discussed somewhere.

**Review by clabby — COMMENTED** [review]
Deployment and changes to drippie contracts look good. Unsure on the configuration, though - was this discussed anywhere? Values don't look bad, just curious if there's a reference to what was decided upon.

**Comment by smartcontracts** [comment]
Specific configuration values are just dummy values for Sepolia. We should agree on real values though.

File: packages/contracts-bedrock/scripts/Artifacts.s.sol
Comment (smartcontracts) [reply packages/contracts-bedrock/scripts/Artifacts.s.sol:1]: Oops

File: packages/contracts-bedrock/periphery-deploy-config/sepolia.json (hunk 2 above)
Comment (smartcontracts) [reply packages/contracts-bedrock/periphery-deploy-config/sepolia.json:21]: Addressing this later after next PR goes thru.

**Review by clabby — APPROVED** [review]
